cmd/mattermost/commands/user.go

```diff
@@ -368,6 +379,57 @@ func userCreateCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func usersToBots(command *cobra.Command, args []string, a *app.App) error {
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	users := getUsersFromUserArgs(a, args)
+	for i, user := range users {
+		if user == nil {
```

Comment:
Can we move the conversion to an App function. I suspect we will want to use this in other places.